ExilenceNextApp/src/utils/object.utils.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable guard-for-in */
+
+function camelToSnake(str: string) {
```

Comment:
Seems like a lot of work is going on here 🤔. Could we perhaps use lodash with `toSnakeCase()` and `toCamelCase()`?